spec/shared_examples/metadata.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
```

Comment:
This file seems to be a duplicate of ` spec/support/shared_examples/metadata.rb`